microsoft-edge/webview2/concepts/threading-model.md

```diff
@@ -69,24 +71,26 @@ private void CoreWebView2_WebMessageReceived(object sender, CoreWebView2WebMessa
 ``` 
 
 > [!NOTE]
-> For WinForms and WPF apps, in order to get full call stack for debugging purpose, native code debugging mode must be enabled.
-> 1.  Enable native code debugging for WebView2 apps.  
->    1.  In your WebView2 project, open the context menu \(right-click\), and choose **Properties**.  
->    1.  Select the **Debug** tab, and ensure the **Enable native code debugging** checkbox is selected, as shown below.
+> For WinForms and WPF apps, to get the full call stack for debugging purposes, you must turn on native code debugging for WebView2 apps, as follows.
+> 1.  Open your WebView2 project in Visual Studio.
+> 1.  In **Solution Explorer**, right-click the WebView2 project and then select **Properties**.  
+> 1.  Select the **Debug** tab, and then select the **Enable native code debugging** checkbox, as shown below.
 
-:::image type="complex" source="../media/webview-enable-native-debug.png" alt-text="Visual Studio Native Code Debugging Configuration Property" lightbox="../media/webview-enable-native-debug.png":::
-   Visual Studio **Debug** Configuration Property 
+:::image type="complex" source="../media/webview-enable-native-debug.png" alt-text="Enabling native code debugging in Visual Studio" lightbox="../media/webview-enable-native-debug.png":::
+   Enabling native code debugging in Visual Studio
 :::image-end:::  
 
 ## Deferrals  
 
-Some WebView2 events read values set on the related event arguments or start some action after the event handler completes.  If you also need to run an asynchronous operation such an event handler, use the `GetDeferral` method on the event arguments of the associated events.  The returned `Deferral` object ensures the event handler isn't considered complete until the `Complete` method of the `Deferral` is requested.  
+Some WebView2 events read values that are set on the related event arguments, or start some action after the event handler completes.  If you also need to run an asynchronous operation, such as an event handler, use the `GetDeferral` method on the event arguments of the associated events.  The returned `Deferral` object ensures that the event handler isn't considered complete until the `Complete` method of the `Deferral` is requested.  
 
-For instance, you may use the `NewWindowRequested` event to provide a `CoreWebView2` to connect as a child window when the event handler completes.  But if you need to asynchronously create the `CoreWebView2`, request the `GetDeferral` method on the `NewWindowRequestedEventArgs`.  After you've asynchronously created the `CoreWebView2` and set the `NewWindow` property on the `NewWindowRequestedEventArgs`, request `Complete` on the `Deferral` object be returned using the `GetDeferral` method.  

```

Comment:
Please check how I reworded the end of the paragraph, instead of "object be returned".  
@tofuandeve @jasonstephen15